src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrapper calendar__wrapper--month-length-30">
+        <div class="calendar__day calendar__day--1 calendar__day--mon"></div>
```

Comment:
calendar__day--mon modifier should be in calendar__wrapper element and use :first-child pseudo class for manage margin for first calendar__day (looks like redundant to make calendar__wrapper element).

```suggestion
      <div class="calendar__wrapper calendar__wrapper--month-length-30 calendar__wrapper--start-day-mon">
        <div class="calendar__day "></div>
```

Reply:
I don't understand how to do it correctly so that everything works as required in the assignment